client/src/app/layout/sidebar.tsx

```diff
@@ -23,6 +23,14 @@ export const SidebarApp: React.FC = () => {
             >
               Trust root
             </NavLink>
+            <NavLink
+              to="/artifacts"
```

Comment:
Could we use our constant `Paths` just like the other elements in the sidebar?

Reply:
good catch @carlosthe19916 I forgot to code after a few months not coding :)